enterprise/admin/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""
+Module imports admin panels needed for Enterprise app.
+"""
+from enterprise.admin.admin import EnterpriseCustomerAdmin, EnterpriseCustomerUserAdmin
```

Comment:
Are we taking advantage of declaring imports here? if not which seems to be the case here then I would suggest reverting/removing the changes for this file.